modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/MoveEntries.es.js

```diff
@@ -103,7 +102,12 @@ MoveEntries.STATE = {
 	 * @review
 	 * @type {string}
 	 */
-	selectFolderURL: Config.string().required(),
+	selectFolderURL: {
+		config: {
+			required: true,
```

Comment:
What's your plan for making this boolean do something?

Reply:
I think it's handled in the `metal-state` implementation here in [State.js](https://github.com/metal/metal.js/blob/master/packages/metal-state/src/State.js#L88-L112). At least that's what my testing/debugging has led me to believe.

Reply:
Makes sense. 👍